### ISSUE TITLE
Conversion start from thumbnail version

### ISSUE THREAD

**Issue by trippo** [outl1ne/nova-media-hub#37]
If I configure nova-media-hub to generate a thumbnail version for all collections, and after I add for each collection a medium conversion. 
When I upload an image, the conversion tool starts from the thumbnail version to crop the medium conversion.
The tool must start every time from the original image, otherwise i obtain photos like this:
Original:
![chiara_s](https://user-images.githubusercontent.com/497169/214542822-8e42bcd4-7dcf-48b2-aa25-fb5b4d829dc6.jpg)
Thumbnail:
![chiara_s thumbnail](https://user-images.githubusercontent.com/497169/214542863-76973914-5a9b-4e52-ab7d-48e65746155a.jpg)
Medium:
![chiara_s medium](https://user-images.githubusercontent.com/497169/214542896-f3152aa5-75b6-4c95-9ea0-d92617e14288.jpg)

**Comment by Tarpsvo** [comment]
Can I see your config please?

**Comment by trippo** [comment]
```
'image_conversions' => [
        '*' => [
            'thumbnail' => [
                // Image format, null for same as original
                // Other options: jpg, pjpg, png, gif, webp, avif, tiff
                'format' => null,
                'width' => 150,
                'height' => 150,
                'fit' => Manipulations::FIT_CROP,
            ]
        ],
        'home' => [
            'medium' => [
                'format' => null,
                'width' => 1200,
                'height' => 1200,
                'fit' => Manipulations::FIT_MAX,
            ]
        ],
        'banner' => [
            'medium' => [
                'format' => null,
                'width' => 1400,
                'height' => 600,
                'fit' => Manipulations::FIT_CROP,
            ]
        ],
  ```

**Comment by Tarpsvo** [comment]
Heya @trippo! The logic was indeed a bit scuffed - all the optimizations were run on the same local file. It should now be fixed in version 1.5.3. Please give it a try and report back. Thanks!
